docusaurus.config.js

```diff
@@ -82,6 +82,12 @@ const config = {
             position: "left",
             label: "Graphcast",
           },
+          {
+            type: "doc",
+            docId: "graphcast/radios/intro",
+            position: "left",
+            label: "Radios",
+          },
```

Comment:
```suggestion
```

Reply:
I think it's better to remove this since `Radios` can just be nested inside the `Graphcast` item from the navbar

Reply:
that makes sense. There's also `Radios` access through the homepage so lmk if you think that should be removed as well - I wanted to provide straightforward access to the Radios but it would make sense for them to click through `Graphcast`

Reply:
updated ✅